.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### Required Information
+
+- Mode (Mock/Sandbox/Live):
+- PayPal card.io-iOS-SDK Version:
```

Comment:
You might specify how to get it, IIRC something like `[CardIOView libraryVersion]`. And is "PayPal" necessary there, or will it confuse?